lib/DBDefs/Default.pm

```diff
@@ -103,6 +103,7 @@ sub WEB_SERVER                { "localhost:5000" }
 # Relevant only if SSL redirects are enabled
 sub WEB_SERVER_SSL            { "localhost" }
 sub LUCENE_SERVER             { "search.musicbrainz.org" }
```

Comment:
Can you remove this and grep for other locations where it's used and removed them too?

Reply:
done